pages/linux/blight.md

```diff
@@ -0,0 +1,24 @@
+# blight
+
+> Utility for changing the display brightness.
+> More information: <https://github.com/gutjuri/blight>.
+
+- Set display brightness:
+
+`blight set`
+
+- Show current display brightness:
+
+`blight show`
+
+- Show maximum display brightness:
+
+`blight max`
+
+- Increase display brightness with %:
```

Comment:
```suggestion
- Increase display brightness in %:
```

what do you think ? is it better ? 🤔

Reply:
Applied changes in [blight: adjust to change requests](https://github.com/tldr-pages/tldr/pull/8633/commits/3f7059197d72071c9d57bd58b73cc50760504011) 🌻